### PR TITLE
React cal poc kl

### LINKED ISSUE #67: Calendar control renders on a page [POC]
[react calendar](https://github.com/wojtekmaj/react-calendar/blob/main/packages/react-calendar/README.md)

Proof of concept for events page

### PR DESCRIPTION
**Pull request by KVinyl**
Changelog from #79 
Moved calendar and event cards to one row.
Allow multiple event cards to be displayed.
Scroll bar is shown when multiple event cards are displayed.
No event cards are shown when selected date has no events.
Text color of day number during the weekend in calendar is white when selected or has event instead of black.

--
Event card text needs formatting.
Does not look good on mobile.
Lots of module warnings during webpack compilation.

### CODE REVIEW

**Comment by KVinyl** [comment]
@readysetagile

**Comment by readysetagile** [comment]
@KVinyl please resolve the merge conflicts and the linting errors

**Comment by KVinyl** [comment]
@readysetagile Merge conflicts and linting errors have been resolved. Did some further refactoring. Of course, more needs to be done, just looking for feedback for I have so far.

**Comment by readysetagile** [comment]
@KVinyl a test is failing.  Make sure you run `npm run build && npm run lint && npm run test` before pushing

**Comment by KVinyl** [comment]
@readysetagile Updated the snapshots to pass testing.

**Comment by KVinyl** [comment]
@readysetagile Further optimization, refactoring, minor bug fixes

**Review by readysetagile — CHANGES_REQUESTED** [review]
It looks like it works great, so nice work there!  

I think that we might want to follow good design patterns here also -- like a MVC.  So let's move the code that is currently in `Events.jsx` into a controller structure, that is, `eventsController.js`  The model is already taken care of in the calendar control.  That way `Events.jsx` becomes the "view", and just handles user events and initialization.  you can ask GPT this question to know what I mean:

> can this react.js code for Events.jsx be refactored so that it follows a view-controller design, that is, the Events.jsx serves as a view to just handle user events and initialization, and the controller does the stuff that happens when an event is fired? [include the code for Events.jsx] 

Seeing this will also likely bring up more refactoring, like being able to configure the link for the calendar through a file and possibly make it data driven also -- along with other maintainability concerns, so let's start with the controller.

**Comment by KVinyl** [comment]
@readysetagile Created two custom hooks. useEvents is responsible for fetching events. useCalendar takes in events as an input and returns the variables/functions used by the Calendar/Events section. 

Fixed unwanted multiple events CSS behavior in Chrome.

File: src/hooks/useEvents.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react';
+import ical from 'cal-parser';
+
+const useEvents = () => {
+  const [events, setEvents] = useState([]);
+
+  const fetchData = async () => {
+    try {
+      const response = await fetch(
+        'https://docs.google.com/document/d/1OcnWWt1qHaJWE-8v_FPtNO8DKviRQ7DMaqylupuvPXE/export?format=txt'
```
Comment (readysetagile) [inline src/hooks/useEvents.js:10]: We will create another backlog item to make this value configurable.  It's outside the scope of this backlog item.

**Review by readysetagile — APPROVED** [review]
Well done, @KVinyl !  This captures the essence of a good design.

**Pull request merged**